### DISCUSSION TITLE
I Can't Apply Labels

### DISCUSSION THREAD

**Discussion by ghost** [community/community#136771] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

![image](https://github.com/user-attachments/assets/8a4d6ebe-620d-4743-8af9-a4c29d120972)
There is no where on the right to click to apply labels. How do I apply labels?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Deekshit7483** [comment, marked as answer]
Hello,

### Answer:
You can't add labels to issues on other people's repositories. You can only add labels to issues in your own repository. If a user creates an issue in your repository, you can add a label to that issue.

![Screenshot from 2024-08-25 20-26-26](https://github.com/user-attachments/assets/b1c9115b-1af7-4cb6-a9aa-f935d89b1cc2)
If anybody creates an issues on your repo, like this you can add an labels.

**Comment by AbuBakkarRaihan** [comment]
Response #2  Brother If you dont have admin or collaborator get admission to with write permissions, you cant add labels. Check when you have permission to publish labels. Reload the page or log in with a exclusive browser to rule out performance problems. Also, verify you are not running on a forked repository, wherein label settings is probably limited.

**Reply by ghost** [reply]
Can you make the first sentence clearer?

**Comment by Yackens** [comment]
As previously indicated, you can't add labels to a repository unless it is open-sourced or you are given to needed permissions to do so. Try again to get those permissions and you WILL be able to add labels. Hope this helps!

**Reply by ghost** [reply]
I'm trying to apply a label that is already in the repository.
